### PR TITLE
Model parallelism: Moving labels to the same device as logits for BridgeTower models

### PR DESCRIPTION
**Pull request by shahad-mahmud**
As suggested in the https://github.com/huggingface/transformers/issues/22561, moving the labels to the same device as the logits are for the BridgeTower models

@sgugger Can you please review?

### CODE REVIEW

**Comment by Leonardbrett** [comment]
My first investment with Charlotte give me profits of over $7,000 and ever since then she has never failed to deliver and I can even say she's the must sincere broker I have ever known'' communicate with her to be a successful investor. On WhatsApp:+𝟭𝟰𝟳𝟱𝟮𝟰𝟮𝟲𝟯𝟯𝟴

**Review by sgugger — APPROVED** [review]
Thanks a lot!
